README.md

```diff
@@ -16,3 +16,78 @@ The project and Python dependencies can be installed from source with:
 ```
 pip3 install .
 ```
+
+## Configuration System
+
+This project provides a configuration system that supplies options (e.g., `user`, `token`) to a command-line interface (CLI) tool.
+The configuration system follows a tiered order, incorporating both files and command-line options.
+
+### Configuration Sources
+
+The configuration system loads options from JSON files located across multiple directories.
```

Comment:
For this first reference to JSON files,
include a link (I usually prefer Wikipedia).